docs/estela/installation/micro-frontends.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: Micro-Frontends Guide
+parent: Installation
+grand_parent: estela
+---
+
+# estela Micro-Frontends Guide (Optional)
+
+estela allows the use of Micro-Frontends only in the user dropdown. This is achieved with the use of [Module Federation](https://webpack.js.org/concepts/module-federation/).
```

Comment:
We should give more context about the aim of using micro frontends. Why are they important? How this helps estela?